src/setup/sign-up-to-mergin-maps/index.md

```diff
@@ -13,13 +13,13 @@ One can view and download public projects even without an account, but creation
 
 1. Navigate to <MainDomainNameLink /> 
 2. Click on **Start for free**
-   ![mergin maps register](./mergin-web-register.jpg)
+   ![Mergin Maps start for free](./mergin-web-register.jpg "Mergin Maps start for free")
 
 3. Click on **Create your free account**
-   ![web create account](./mergin-web-create-account.png)
+   ![Mergin Maps create free account](./mergin-web-create-account.jpg "Mergin Maps create free account")
```

Comment:
This is a good place to talk about workspaces:
 - after registration, you create a new workspace if you do not have any invitation
 - the workspace will have free tier and can be upgraded,.. (+ fix also the above [line 6](https://github.com/MerginMaps/docs/pull/280/files?file-filters%5B%5D=.md&show-deleted-files=true&show-viewed-files=true#diff-87df9466860ba3e3124e56dd9b0af66262f3b49a169bff8bde0ee602865b2fddR6) - it says that you will get tier for your account, should be workspace instead)